pkgs/games/tuxtyping/default.nix

```
@@ -0,0 +1,22 @@
+{ stdenv , fetchurl, pkgconfig, SDL, SDL_image, SDL_mixer, SDL_ttf, librsvg }:
+
+stdenv.mkDerivation rec {
+  version = "1.8.0";
+  name = "tux-type-${version}";
+
+  src = fetchurl {
+    url = "https://sourceforge.net/projects/tuxtype/files/tuxtype-source/TuxType%20${version}%20-%20Source/tuxtype_w_fonts-${version}.tar.gz/download";
+    sha256 = "078jvjizycra4b7wp784gc19jnn4jm125939pqip1pwn5rm95c20";
+  };
+
+  patchPhase = ''
+    patchShebangs data/scripts/sed-linux.sh
+  '';
+
+  nativeBuildInputs = [ pkgconfig SDL SDL_image SDL_mixer SDL_ttf librsvg ];
```

Dependencies that are run during build should go to `nativeBuildInputs`, runtime dependencies go to `buildInputs`:

```suggestion
  nativeBuildInputs = [ pkgconfig ];
  buildInputs = [ SDL SDL_image SDL_mixer SDL_ttf librsvg ];
```